### DISCUSSION TITLE
Installing Astroid 2.6.2 bring success screen with old Joomdev Links & Logo?

### DISCUSSION THREAD

**Discussion by Chacapamac** [templaza/astroid-framework#229] [General]
**Just test installing Astroid 2.6.2 bring success screen with old Joomdev Links & Logo?**

_**Just a note: The JD Builder** link to a Joomdev page with, it seems, a newer version of JD Builder (1.11)  Anybody test that? (even if the JD Builder wasn't to great anyway. It will be interesting to know if it will work with the NEW Astroid and can update existing websites...._
—> https://www.joomdev.com/jd-builder-1-11/?utm_campaign=astroid_install_screen

![Astroid-2 6 2-install](https://user-images.githubusercontent.com/1907276/197240004-fee86095-170c-483a-afad-84c47206307c.jpg)

**Comment by sonvnn** [comment]
This is old article of Joomdev. For JD Builder update please contact to Stackideas Team. 

Thanks & Best Regards,
Sonny

**Reply by Chacapamac** [reply]
Thanks @sonvnn  for the JD BUILDER info's.
You should change the links for the NEW Astroid stuff. That can be  confusing for new users.

Good day to All!
